packages/webapp/src/utils/get-build-config.js

```diff
@@ -0,0 +1,50 @@
+import { handleEvents } from './azure-signalr.js'
```

Comment:
Good idea to extract the upload config for the metric file, but I think it would be better to make it file type agnostic. As METRIC_UPLOAD_TYPE is hard coded, it would be better to make getBuildConfig to accept a file upload type so that it can be reused elsewhere in the application, especially as the name of the file is file type agnostic